migtests/lib/yb.py

```diff
@@ -78,3 +79,40 @@ def get_objects_of_type(self, object_type, schema_name="public") -> List[str]:
 		cur.execute(f"select relname from pg_class join pg_namespace on pg_class.relnamespace = pg_namespace.oid"+
 			f" where nspname = '{schema_name}' AND relkind = '{object_type}'")
 		return [obj[0] for obj in cur.fetchall()]
+	
+	def get_sum_of_column_of_table(self, table_name, column_name, schema_name="public") -> int:
+		cur = self.conn.cursor()
+		cur.execute(f"select sum({column_name}) from {schema_name}.{table_name}")
+		return cur.fetchone()[0]
+		
+	def get_count_index_on_table(self, schema_name="public") -> Dict[str,int]:
+		cur = self.conn.cursor()
+		cur.execute(f"SELECT tablename, count(indexname) FROM pg_indexes WHERE schemaname = '{schema_name}' GROUP  BY tablename;")
+		return {tablename: cnt for tablename,cnt in cur.fetchall()}
+
+	def get_distinct_values__of_column_of_table(self, table_name, column_name, schema_name="public") -> List[Any]:
```

Comment:
Do you really intend to use double underscores in the function name?

Reply:
Oh no, it must be by mistake.